introduction_to_amazon_algorithms/forecasting_services_comparison/sagemaker_example.ipynb

```diff
@@ -0,0 +1,1543 @@
SupXhO/2siJhWa8gttDoptt+rBTkb/o5l92vtGqdUlI8qSnyzrlfCMuPPP2F9+snI3+KunZi+/Qp3jZezvXOx33ID/hNPSfB3BjT7lKHP4O92apvH5i89DOekX1HM+lR0eeAgFEcuxlZBMsCiRRJqjoPQxEkEDjsiMdCLgeXlf2H+7hua73soqaeq6YfvsN91a0S8ZMsWkT59YvQGBJHa/fbgzkIcRvTuZO3dXRlbnNzvetcugdpakfXrdYLklmqGtHNHXJTgevBR6uentkvJBOaP3teE81qr2yt+QvZBKKbU1QFPr8toGJei4pUEcp9+WdMYOwSKgmHzRkIDBlJVpSlbxzIETTVfkbP5DmhJdAVKD8fb7XSEYHoPWKWkVBNUa+nL7NdPcxRobS3TkUhWSc478WgsLzzXIcewPv8MhVPGIVQn6kLk56vU1+/dIDlVJdn25GPYH7g74XMh2IDk240QEyi5+96It/u5GR0zLN4n7ShPuk5hoUpdXUf3JKsER4+j8bW3dW3ZwvB2v4DmoZm/5xyrz6Vg0Syah79I06g3KJ5Xhm1bcgaQ0rUb7htvxXXX/ZHPDAt/w/TpR5G/xT27NStKs4nSUhWj7EXYtRu/n5RtBno9yeKe3Qgpen/t669N2rKVsP/SwwkWzgBJv11NDyUlYbp1zLhtNu1dkUbkxLhIa79paGE0VFZq+4jVOghO2zcrbRd/l5Pw9L0p7XqqMY9Q7hhd28kOhcGAkuNI6qjgP+RQfEcfu3fH8BeyD5LDsFgsdOrUCdtfymp/GsJCSskyzBntY3s5tofuw7B7OwaDGhHGy115FvmLZiTdzjr3HzjOiad/qaWlhIaNwLgwSqkWBDjiiBBGIwiyH3PFW1l5zu4t2Lfej6UysbIq1NTguOAspCT9X7GQ+/ZH3LE9TqApuiMJ5/R1+FoJOoS9kuMqG4KgLxvaAmnDeqxz/4HgSq0mH4bjmis52/VkXCUZVdatxIQGD6X+u58IzJiZcp8RG6g0k2qbXeCYf7zPCuWGjMbaXoTp1mFYdj6PfdOtCesZ5/+A6evk4kaRIHlrYgW8YdxXuAfoK0tnCqGpAYfSRP9FbyUwjU3zvsd+z+2R38f04zzyTjke6uoYoG7A+sZr5B15CKLO2Az1v1H484gEu7FsoBrykbzbkdxJVDxFEc91NxA4MJpV79ZNpVs3hUWLJCxvvEbhyEHkXH05+Qftl/Q40s4dBKbNwHf2+ahJaGFqfj7B6TMIlzm2bBHilK1Fbzm28kcRg86Mz0+xlHHHZ/9gRWVy6lyYit3cnDjhVjppfeSOK+Lt8NSCQoSmJl3V4WxgXPir1nvZmuYoWWge8TKBTkel3F4xd0Fyb0DyZMfecD3wCM7Ff75wl7hrJ4LXi9xfC5Jb9yN7+t1C3ZQVUYsZVaFowVDsW+7NYOciSpduEfGiP0O8Kzx/jws6gkGM875PKWSXDYJjNHeDcDAQi8LCjq2m6h4/hU+y4HKh5jgSPlfNnaif/Av+zkdHPgsU749i6Z7RMeUW8b5Yb+DWKCpScTo7yHJQjapbq8XFWqI8ieYIgKCGkO3Jg+jW8HU5CdnaC8fKMxCUAO7e1xAsSEHnFQQ8V1wTF6hbX/4XOXdE32/h60spKqa0VOVm7mbnqVdz221m9t/fppvTVlX9nmTPNdenFCpTzF0y679GEwp1D7gLc9WH2Lbcl9E2VqtWpIlLaPn9lPTshO2fT6fc1rhQS1aGE5oVFQK5uWrc3CDvhKOwvJyalROLwl9GYt+YGRvWvulWcldnlvxpD9T8fMQkvRW+s8/Hd/b5e30M/9+R9ZvE7Xbz0Ucfcd9993HjjTdy33338dFHH+Fu1dv3FzoeodFjkMu64zup7VUkw+pV2B+6D2qd5OZGM/r+LsfjKzsn6XbGhb9g0AnO3Tfeivs6/eybseFXHBuuQ/RlrhS8t1A/6RdcAxOzuKbvv8HywXuaAEIayP0HIKgq0hb9DKK54l0cK+MTCT16KDQ3C8S0caaFYc0qcu68BbFCn6KaMC6LlToK44Lk5uEv4Ol7feLKRiOhEaPS0nTCBvcJdORWsNtVNuwZiMO3CJT0Qk3tRWu6teTeqBswWp97CtvD9yd8HobqyEUpLkEqT6wkS/4KTHX6fUCZQummTQxrgvmsXh3/mFU6d0YIBhGcWuAnNGmqpzsb8wnmaWwG068/I+j5i0pWgrljUYzpxVqSQhDJXXUGlgodOn4LvOdfTOCAg+I+mzBBZtEiCaG6GqliDxgNmrJ3Em6ruHMnSrdu5P7tJCz/1u+x9557IY3vaj2MiqIpwffsGd2fwbUO+5Z7ELOxkhMEPllzBmoouUJyOM51uXSC5M5dCI4YRWBmfF+s8cd5FA/rh3HpH5mPRQfBMePwnXZGwueGht8p+HV82n5jQfaSt/JUTDX6faFJEQggrV4FrswpkR0BwyYtGSP3H0BVVbyXqeTeSM7aK+LVrAURV/+78XU5IaP9N3z0Oc0Pa1Y5f4YNVNhaLLaSLO7ZjaCqEfq3HgwNv2tUZFVNmzgOjRiFajbrBsn5+ez1SnIqCyjB5UK1JzJDTNWfk7vilDh9CFv54xT8OjZO0DIZ5L79cM7/LWXbUzhI7gg6fbhabjKB/dYbyTt2Tsr1c1eehn3zHRnvP1i0P8H8CRjcG1ClHLy9riKUm523vdyrF+LunZGsheisQzUaUXPzKClR+IbZ/GGbzhtvGPF6Bd0EQvhZ17p1OTh5ipagTAJv76twD8wgURUDU+23mGq/yrgVJMEr2WxGyctP25PsP1SjvTsuOBvQguQuXWLa10IhTPO+z6i3ObLPTscQzMtMZM7T+xqaRrSPbZYJlC5dUZP0jOfccA2mb7/a62P4/46s3iRffvklM2fO5IYbbuCVV17hgw8+4JVXXuGGG25g5syZfPXVXz/Y3oTnymtxLlmN0A7VDqmlp26b0jNO6ClQNAtf1+TBt7R5E7Ke5+zM/Qnp9U4C/k5HUzt
```

Comment:
Change the kernel from mxnet to a python3 one, such as `conda_python3` , in both notebooks.